src/main/include/subsystems/Shooter.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <rev/CANSparkMax.h>
+#include <ctre/phoenix/motorcontrol/can/TalonSRX.h>
+#include <units/math.h>
+#include <frc/controller/PIDController.h>
+#include <frc/DigitalInput.h>
+
+#include "Constants.h"
+
+class Shooter
+{
+    public:
```

Comment:
Indentation: `public:` (and all access specifiers) should be aligned with `class`.

```suggestion
public:
```